music-suite.cabal

```diff
@@ -66,7 +66,8 @@ library
                         random,
                         neat-interpolation,
                         modular-arithmetic,
-                        random-shuffle
+                        random-shuffle,
```

Comment:
This does not seem to be used.

Reply:
It looks like it's in Score/Internal/VoiceSeparation.hs
https://github.com/hanshoglund/music-suite/blob/6e7be0277e85297c38496ec56a36fbc853620feb/src/Music/Score/Internal/VoiceSeparation.hs#L37

Reply:
My bad, was confused by the diffs.